src/arc.directive.ts

```diff
@@ -16,7 +17,7 @@ function createDirectionCapture(direction: Direction, target: HTMLElement) {
   if (!target) {
     throw new Error(
       `Cannot set [arc-${Direction[direction]}] to an undefined element!` +
-        'Make sure the element you\'re passing is defined correctly.',
+        "Make sure the element you're passing is defined correctly.",
```

Comment:
Just curious why use double quotes instead of single?

Reply:
This is prettier's formatting -- it optimizes for least number of escape sequences in a string.